<?xml version="1.0"?>

<launch>
<group ns="H01">
	<!-- <include file="$(find pub_sub)/launch/pub_sub.launch" /> -->
    <param name="vehicle_name" value="H01" />
    <param name="print_rate" value="0.5" />
	<rosparam command="load" file="$(find marble_status_monitor)/config/topic_list.yaml" />
	
	<node name="status_monitor_node" pkg="marble_status_monitor" type="status_monitor_node" output="screen" />
</group>
</launch>
